x-pack/plugins/apm/public/components/app/TransactionOverview/index.tsx

```diff
@@ -154,9 +158,34 @@ export function TransactionOverview() {
               <h3>Transactions</h3>
             </EuiTitle>
             <EuiSpacer size="s" />
+            {!transactionListData.isAggregationAccurate && (
+              <EuiCallOut
+                title={i18n.translate(
+                  'xpack.apm.transactionCardinalityWarning',
+                  {
+                    defaultMessage:
+                      'This view shows a subset of reported transactions.',
+                  }
+                )}
+                color="danger"
+                iconType="alert"
+              >
+                <p>
+                  The number of unique transaction names exceeds the configured
+                  value of {transactionListData.bucketSize}. Try reconfiguring
+                  your agents to group similar transactions or increase the
+                  value of
+                  <EuiCode>
+                    xpack.apm.ui.transactionGroupBucketSize
+                  </EuiCode>.{' '}
+                  <EuiLink href="#">Learn more in the docs.</EuiLink>.
```

Comment:
I think we should add the docs link instead of `#`.

Reply:
oooh, good catch!

Reply:
I guess still waiting for https://github.com/elastic/kibana/issues/67691